src/test-utils.tsx

```diff
@@ -1,29 +1,51 @@
 import * as React from 'react';
 import ApolloClient from 'apollo-client';
+import { DefaultOptions } from 'apollo-client/ApolloClient';
 import { InMemoryCache as Cache } from 'apollo-cache-inmemory';
 
 import { ApolloProvider } from './index';
-import { MockLink } from './test-links';
+import { MockedResponse, MockLink } from './test-links';
 export * from './test-links';
 
-export class MockedProvider extends React.Component<any, any> {
-  static defaultProps = {
+interface IProps {
```

Comment:
Prefer `export interface MockedProviderProps` - both for reuse/composition and better IDE suggestions